github/ci/prow-deploy/files/jobs/kubevirt/kubevirt.github.io/kubevirt-github-io-postsubmits.yaml

```diff
@@ -0,0 +1,55 @@
+postsubmits:
+  kubevirt/kubevirt.github.io:
+  - name: push-kubevirt.github.io-master-build-and-push-to-gh-pages
+    branches:
+    - ^master$
+    annotations:
+      testgrid-create-test-group: "false"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
```

Comment:
@fgimenez TBH I have not the slightest idea what this annotation is for? Is this some kind of reuse directory where the docker images are somehow stored magically? Maybe the docker image cache writes there?

Reply:
Sorry for the late response, this preset mounts a hostPath in `/var/lib/stdci/shared/kubevirt-images/` which is used for RHEL and windows tests, from kubevirt/kubevirt:
```
$ grep -r stdci .
grep: ./.git/objects/pack/pack-0304ad5d57e7cbd19c81fbb4e663a86626fa821d.pack: binary file matches
./automation/test.sh:export RHEL_NFS_DIR=${RHEL_NFS_DIR:-/var/lib/stdci/shared/kubevirt-images/rhel7}
./automation/test.sh:export RHEL_LOCK_PATH=${RHEL_LOCK_PATH:-/var/lib/stdci/shared/download_rhel_image.lock}
./automation/test.sh:export WINDOWS_NFS_DIR=${WINDOWS_NFS_DIR:-/var/lib/stdci/shared/kubevirt-images/windows2016}
./automation/test.sh:export WINDOWS_LOCK_PATH=${WINDOWS_LOCK_PATH:-/var/lib/stdci/shared/download_windows_image.lock}
``` 
I think we can safely remove the preset for the rest of jobs.